mesonbuild/compilers/mixins/gnu.py

```diff
@@ -491,6 +491,9 @@ def get_compiler_dirs(self, env: 'Environment', name: str) -> T.List[str]:
                 return self._split_fetch_real_dirs(line.split('=', 1)[1])
         return []
 
+    def get_legal_code_compiler_args(self) -> T.List[str]:
+        return []
+
```

Comment:
Don't we already inherit the same definition from the parent class? (`class GnuLikeCompiler(Compiler)`)